docs/content/Getting-Started/Cloud/05-Add-a-pre-aggregation.mdx

```diff
@@ -0,0 +1,47 @@
+---
+title: Add a pre-aggregation
+permalink: /getting-started/cloud/add-a-pre-aggregation
+category: Getting Started
+subCategory: Cube Cloud
+menuOrder: 6
+isDisableFeedbackBlock: true
+---
+
+From the Developer Playground, execute a query:
+
+<div style="text-align: center">
+  <img
+    alt="Cube Cloud Deployment Developer Playground Screen showing query results"
+    src="https://ucarecdn.com/d2210bb7-a35e-4da7-ba29-447da24cc06d/"
+    style="border: none"
+    width="80%"
+  />
+</div>
+
+Just above the results, click on **Query was not accelerated with
```

Comment:
Let's put an emphasis on the query latency. It's three-digit in ms, right?

Reply:
Can we discuss this in a Slack huddle?